app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def increment_posts_counter
+    user = User.find(user_id)
+    user.update(posts_counter: user.posts_counter + 1)
+  end
+
+  def recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).first(5)
+  end
```

Comment:
- [ ] A more effective query would be 

```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

The main idea is that instead of `first` we use `limit`.  Remember that the comments have been already associated with the posts.